.github/workflows/test.yml

```diff
@@ -7,6 +7,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+# env:
+#   store: /home/runner/nix
```

Comment:
This isn't working correctly on Mac, I think it should be fine on the ubuntu runner though. Need to look into this later.